### PR TITLE
Grammar

### PR DESCRIPTION
**Pull request by 1griffy**
## Description
Cost of execution isn't a resource

### CODE REVIEW

File: docs/book/src/index.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Whilst Rust is a great systems programming language (and Sway itself is written
 
 Rust shines because it can use zero-cost abstractions and its sophisticated borrow-checker memory model to achieve impressive runtime performance for complex programs without a garbage collector.
 
-On a blockchain, cost of execution and deployment is the scarce resource.
+On a blockchain, cost of execution and deployment is scarce.
```
Comment (sdankel) [inline docs/book/src/index.md:33]: This suggestion doesn't really make sense either. The cost isn't scarce, it's just costly.

```suggestion
On a blockchain, execution and deployment are limited by transaction costs.
```

File: docs/book/src/index.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ Whilst Rust is a great systems programming language (and Sway itself is written
...
 Memory usage is low and execution time is short.
```
Comment (sdankel) [inline docs/book/src/index.md:34]:
```suggestion
Memory is an expensive resource.
```

Comment (IGI-111) [reply docs/book/src/index.md:34]: You're losing key information and this isn't what I mean here.

File: docs/book/src/index.md (hunk 1 above)
Comment (IGI-111) [reply docs/book/src/index.md:33]: There may be a better term than cost to analogize with space and time, which are resources.

File: docs/book/src/index.md (hunk 2 above)
Comment (1griffy) [reply docs/book/src/index.md:34]: Updated

**Comment by IGI-111** [comment]
Yeah, I'm just going to close this, the new formulation isn't really that much clearer and it's doing other random changes now. We don't have the time to spend on insubstantial documentation changes unless they're no-brainers like bulk typo fixes.

**Pull request closed without merging**